third-party/linshare/src/test/resources/docker-compose.yml

```diff
@@ -16,9 +16,11 @@ services:
         links:
             - database:linshare_database
             - mongodb:linshare_mongodb
+            - smtp:linshare_smtp
```

Comment:
you cannot use james smtp?

Reply:
This third party project do not depend on a james product hence this would technically be impossible - at least as long as we are not writing integration tests - not the case here.

Also, we will aim at checking that the side effect we hope for (sharing with linshare) is performed. We can ensure this by browsing the sharing API.

We would thus never need a real configuration for the smtp - we don't care about LinShare internal mechanism IMO. So I would argue that even on integration tests this might not be desirable. Let's not enforce too much LinShare internal mechanisms...

Also please note that having a dockerized service connecting to a host service might not be trivial, the ip address affected to the James server varying... Better keep things simple...